site/content/blogs/apprunner-vpc.ja.md

```diff
@@ -0,0 +1,177 @@
+![App Runner](./../assets/images/apprunner-vpc-blog/apprunner-arch.png)
+
+Service を作成すると、裏では App Runner が所属する VPC で [Amazon Elastic Container Service (ECS)](https://aws.amazon.com/jp/ecs/) によってオーケストレーションされた [AWS Fargate](https://docs.aws.amazon.com/ja_jp/AmazonECS/latest/userguide/what-is-fargate.html) タスクとして Application コンテナがデプロイされます。デフォルトでは、Application によって開始されたすべてのアウトバウンドトラフィックは、NAT ゲートウェイとこの App Runner が所有する VPC にプロビジョニングされたインターネットゲートウェイを経由してインターネットにルーティングされます。従ってこのモードでは、App Runner でホストされている Application は、インターネット上のパブリックエンドポイントにのみ接続できますが、VPC 内のプライベートエンドポイントに到達することはできません。
+
+![App Runner Deep Dive](./../assets/images/apprunner-vpc-blog/apprunner-deepdive.png)
+
+[App Runner サービス の VPC サポートの発表](https://aws.amazon.com/jp/about-aws/whats-new/2022/02/aws-app-runner-amazon-vpc/)により、アプリケーションは Amazon RDS データベースや Amazon ElastiCache クラスタなど、VPC 内でホストされているプライベートリソースに接続できるようになりました。
+
+VPC の内部で何が起こっているかを理解するには、[AWS App Runner の VPC ネットワーキングに関して Dive Deep した記事をチェック](https://aws.amazon.com/jp/blogs/containers/deep-dive-on-aws-app-runner-vpc-networking/)してください。要約すると、VPC コネクタを作成することで、App Runner サービスは、アウトバウンドトラフィックのみに使用できる一方向の Elastic Network Interface (ENI) を所有している VPC 上に作成します。(つまり、ENI の IP アドレスにインバウンドリクエストを送信することはできません。) この ENI には、カスタマイズしたアウトバウンドルールを含むように変更できるセキュリティグループがあり、希望する宛先エンドポイントへの通信を許可します。
+
+## アーキテクチャ例
+
+AWS Copilot CLI と AWS App Runner の VPC サポートの機能を実証するために、以下の図のようなアーキテクチャを実装する方法をご紹介します。
+![Blog Demo Architecture](./../assets/images/apprunner-vpc-blog/blog-arch.png)
+
+必要なステップは以下の通りです。
+
+- Copilot の環境概念を使用して、Customer VPC を作成します。
+- この VPC 内のプライベートサブネット内に [Amazon Aurora](https://aws.amazon.com/jp/rds/aurora/) PostgreSQL データベースを作成します。
+- AWS App Runner 上で動作する Service を、VPC 内にあるデータベースに VPC コネクターを使用して接続し、適切なセキュリティグループのルールを設定します。
+- Python 向けの一般的な ORM である SQLAlchemy を使用してデータベースに接続し、接続がうまくいくことを確認します。
+
+### 前提条件
+
+上記のアーキテクチャをビルドするためには、以下の前提条件が必要です。
+
+- [AWS アカウント](https://aws.amazon.com/jp/premiumsupport/knowledge-center/create-and-activate-aws-account/)と、上記のインフラストラクチャをデプロイするリージョンを選択していること。
+- ローカルワークステーションに [AWS Copilot CLI](https://aws.github.io/copilot-cli/ja/docs/getting-started/install/) がインストールされていること。
+- [AWS CLI](https://docs.aws.amazon.com/ja_jp/cli/latest/userguide/cli-configure-quickstart.html#cli-configure-quickstart-config) を使用して、または[環境変数](https://docs.aws.amazon.com/ja_jp/cli/latest/userguide/cli-configure-envvars.html)を使用して、[AWS 認証情報](https://aws.github.io/copilot-cli/ja/docs/credentials/)を適切に構成している。
+- ローカルのワークステーションに [Docker](https://www.docker.com/products/docker-desktop/) がインストールされ、実行されている。
+- 次のサンプルリポジトリをクローンしている。
+  - `git clone https://github.com/aws-rafams/apprunner-vpc-copilot`
+
+### Application と Environment の作成
+
+まず、関連するサービス、環境、パイプラインの論理的なグループを作成する必要があります。AWS Copilot の用語では、これを _Application_ と呼びます。
```

Comment:
I think we should use Service, Environment, Pipeline because these words here is Copilot's concept.